.github/workflows/build_images.yaml

```
@@ -29,19 +29,29 @@ jobs:
       matrix:
         image: ${{ fromJson(needs.setup.outputs.images) }}
     steps:
+    - name: Free up additional space
+      run : |
+        sudo rm -rf /usr/share/dotnet
+        sudo rm -rf "$AGENT_TOOLSDIRECTORY"
```

maybe better:

```suggestion
    - uses: jlumbroso/free-disk-space@v1.3.1
      with:
        docker-images: false
        large-packages: false
```

or

```suggestion
    - uses: jlumbroso/free-disk-space@v1.3.1
      with:
        tool-cache: true
        docker-images: false
        large-packages: false
```

---

:+1:  - I didn't realise there was an action now to handle this
I'll put in another commit once we get `test.harbor` back up to give us a target to push to